core/src/impl/Kokkos_Default_Graph_Impl.hpp

```diff
@@ -45,6 +45,8 @@ struct GraphImpl : private ExecutionSpaceInstanceStorage<ExecutionSpace> {
       GraphNodeImpl<ExecutionSpace, Kokkos::Experimental::TypeErasedTag,
                     Kokkos::Experimental::TypeErasedTag>;
 
+  using aggregate_impl_t = GraphNodeAggregateDefaultImpl<ExecutionSpace>;
```

Comment:
I don't necessarily see an issue with this but did you mean to change the access specifier to public?

Reply:
It's already public for the `Cuda`, `HIP` and `SYCL` backends. I just made it consistent (it's in the PR description :wink:).